src/data/reducers/index.js

```diff
@@ -1,13 +1,15 @@
 import { combineReducers } from 'redux';
 
-import assets from './assets';
+import { assets, metadata, request } from './assets';
```

Comment:
The changes made to this file is just a reflection of the changes to reducers/assets.js.